interop/Interop/Client/TestCase/CancelAfterBegin.hs

```diff
@@ -5,19 +5,19 @@ import Network.GRPC.Common
 
 import Interop.Client.Connect
 import Interop.Cmdline
-import Interop.Util.Exceptions
 
 import Proto.API.Interop
 
 -- | <https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md#cancel_after_begin>
 --
 -- This is not really testing anything about the server, but rather about how
--- cancellation gets reported by the grapesy client library itself.
+-- cancellation is handled by the grapesy client itself. In particular,
+-- immediate cancellation does not result in any client-side exceptions or
+-- errors.
 runTest :: Cmdline -> IO ()
 runTest cmdline =
-    withConnection def (testServer cmdline) $ \conn -> do
-      assertThrows (assertEqual GrpcCancelled . grpcError) $
-        withRPC conn def (Proxy @StreamingInputCall) $ \_call ->
-          -- Immediately cancel request
-          return ()
```

Comment:
This change is not conform the spec, I think (see the interop test referenced in the test description). Let's discuss.